src/main/java/htsjdk/samtools/cram/ref/ReferenceSource.java

```diff
@@ -63,7 +62,7 @@ private ReferenceSource() {
     }
 
     public ReferenceSource(final File file) {
-        this(file == null ? null : file.toPath());
+        this(IOUtil.toPath(file));
```

Comment:
IMO the original code was clearer, since `null` is a directly supported case in the constructor.